docs/articles/documentation/using-testcafe/common-concepts/screenshots-and-videos.md

```diff
@@ -0,0 +1,308 @@
+---
+layout: docs
+title: Screenshots and Videos
+permalink: /documentation/using-testcafe/common-concepts/screenshots-and-videos.html
+---
+# Screenshots and Videos
+
+TestCafe allows you to take screenshots of the webpage under test and record videos of test runs.
```

Comment:
TestCafe allows you to take screenshots of the tested webpage and record videos of test runs.